draft-ietf-quic-multipath.md

```diff
@@ -642,76 +626,75 @@ indicated as "standby" by the peer or non-usage of some locally available paths.
+own local logic if the path should be used.
+
+If an endpoint starts using a backup path
 because it has detected issues on the paths marked as "available", it is RECOMMENDED
 to update its own path state signaling such that the peer avoids using the broken path.
 An endpoint that detects a path breakage can also explicitly close the path
-by sending a PATH_ABANDON frame (see {{path-close}}) in order to avoid
-that its peer keeps using it and enable faster switch over to a standby path.
+by sending a PATH_ABANDON frame (see Setion {{path-close}}) in order to avoid
+that its peer keeps using it and enable faster switch over to a backup path.
 If the endpoints do not want to close the path immediately, as connectivity
 could be re-established, PING frames can potentially be used to quickly detect
 connectivity changes and switch back in a timely way.
 
-If no frame indicating a path usage preference was received for a certain path,
-the preference of the peer is unknown and the sender needs to decide based on it
-own local logic if the path should be used.
-
-Endpoints use the Path ID
-in these frames to identify which path state is going to be
-changed. Note that both frames can be sent via a different path
-and therefore might arrive in different orders.
-The PATH_AVAILABLE and PATH_BACKUP frames share a common sequence number space
-to detect and ignore outdated information.
+The PATH_AVAILABLE and PATH_BACKUP frames share a common, per-path sequence number space
+to detect and ignore outdated information, as further described in {{path-backup-available-frame}}.
+This is needed as they might arrives in different orders,
+e.g if sent via different paths.
 
 ## Path Close {#path-close}
 
-Each endpoint manages the set of paths that are available for
-transmission. At any time in the connection, each endpoint can decide to
-abandon one of these paths, for example following changes in local
-connectivity or local preferences. After an endpoint abandons
-a path, the peer can expect to not receive any more packets on
-that path.
-
-Note that other explicit closing mechanisms of {{QUIC-TRANSPORT}} still
-apply on the whole connection. In particular, the reception of either a
-CONNECTION_CLOSE ({{Section 10.2 of QUIC-TRANSPORT}}) or a Stateless
-Reset ({{Section 10.3 of QUIC-TRANSPORT}}) closes the connection.
+At any time in the connection, each endpoint can decide to
+abandon a paths, for example following changes in local
+connectivity or local preferences.
+An endpoint that wants to abandon a path MUST explicitly
+close the path by sending a PATH_ABANDON frame (see {{path-abandon-frame}}).
+This is true whether the decision to close the path results
+from implicit signals such as an idle time-out (see {{idle-time-close}})
+or packet losses as well as for any other reason such as management
+of local resources.
 
-An endpoint that wants to close a path MUST explicitly
-terminate the path by sending a PATH_ABANDON frame (see {{path-abandon-frame}}).
```

Comment:
Can we keep that text? It says something very clearly -- that there is no other way to close a path. That's the main message of the "close path" section.